assets/challenges.html

```diff
@@ -35,29 +38,52 @@ <h1 class="title">code.me </h1>
                 </div>
             </nav>
         </div>
-
-        <div class="hero-body">
-            <div class="container has-text-centered">
-                <div class="column is-6 is-offset-3">
-                    <h3 class="subtitle" id=questionText>
-                        Please select a level of difficulty for the daily challenge </h3>
-       
-            <div class="container">
-                <div class="column is-8 is-offset-2">
-                    <div class="buttons">
-                        <button class="button is-rounded is-primary is-medium" id="easy">
-                            Easy
-                        </button>
-                        <button class="button is-rounded is-warning is-medium" id=medium>
-                            Medium
-                        </button>
-                        <button class="button is-rounded is-danger is-medium" id="hard">
-                            Hard
-                        </button>
+        
+                     
+                    <div class="container column is-12">
+                        <div class="hero-body">
+                            
+                                <div class="column is-6 is-offset-3">
+                                    <h3 class="subtitle" id=questionText>
+                                        Please select a level of difficulty for the daily challenge </h3>
+    
+                                    <div class="container">
+                                        <div class="column is-8 is-offset-2">
+                                            <div class="buttons">
+                                                <button class="button is-rounded is-primary is-medium" id="easy">
+                                                    Easy
+                                                </button>
+                                                <button class="button is-rounded is-warning is-medium" id=medium>
+                                                    Medium
+                                                </button>
+                                                <button class="button is-rounded is-danger is-medium" id="hard">
+                                                    Hard
+                                                </button>
+                                            </div>
+                                        </div>
+    
+                                    </div>
+                                </div>
+                                    <div class="container">
+                                    <h3 class="subtitle" id=questionText>Top 5 Related Videos: </h3>
+                                    <div
+                                    style="height:300px;width:220px;border:1px solid #ccc;font:16px/26px Georgia, Garamond, Serif;overflow:auto;">
```

Comment:
This is fine for now but we would need to take the in-line styling out and have it added in a CSS file once we being working on that.